dev/tools/gen_defaults/lib/bottom_sheet_template.dart

```diff
@@ -24,6 +24,9 @@ class _${blockName}DefaultsM3 extends BottomSheetThemeData {
 
   @override
   Color? get surfaceTintColor => ${componentColor("md.comp.sheet.bottom.docked.container.surface-tint-layer")};
+
+  @override
+  BoxConstraints? get constraints => const BoxConstraints(maxWidth: 640.0);
```

Comment:
Should we talk to the Tokens team about tokenizing this, since the value appears in the spec (https://m3.material.io/components/bottom-sheets/specs#3b256f38-c4fd-44e5-ba46-c5858b626ec7)?

Reply:
We talked last year about it (b/254925602) but all they're considering for now is spacing.

Reply:
Ideally, everything would be a token!